x-pack/plugins/infra/public/pages/link_to/use_host_ip_to_name.ts

```diff
@@ -25,6 +25,7 @@ export const useHostIpToName = (ipAddress: string | null, indexPattern: string |
             });
             setLoadingState(false);
             setData(response.data);
+            setError(null);
```

Comment:
How about we move this before the `try` just after the `setLoadingState`? Otherwise the loading indicator is swallowed by the error message after an initial error.